src/lua/tags.c

```diff
@@ -280,6 +280,16 @@ int dt_lua_tag_get_tagged_images(lua_State *L)
   return 1;
 }
 
+static int dt_lua_tag_get_path_images(lua_State *L)
```

Comment:
The name implies that a table of images is being returned, but actually a reference count is being returned.  Perhaps naming it tag_get_reference_count would better reflect what the function does.

Reply:
good point, thank you